lit_tests/kernel/wave/expansion.py

```diff
@@ -88,21 +88,21 @@ def test_read_write_equal_sizes():
         # CHECK: %a
         # CHECK-NEXT: %c
         # CHECK-NEXT: %read_0_0
-        # CHECK-SAME: (%a, 4)
+        # CHECK-SAME: (%a, 4, None)
```

Comment:
I thought we didn't need to print None anymore?

Reply:
We don't print `None`s when we are pretty-printing `CustomOp`s, but those tests are printing IR in `%read_3 : [num_users=1] = call_function[target=shark_turbine.kernel.ops.wave_ops.read](args = (%allocate_1, 4, None, None), kwargs = {})` form.

Reply:
Ah makes sense. Thanks!